File: mob/src/app/transporte/principal/pessoa.filtro.dto.ts
import { FiltroDto } from 'src/app/comum/transporte/filtro';

export class PessoaFiltroDto extends FiltroDto {

    constructor(
        public pesq?: string[],
        public nome?: string[],
        public cpf?: string[],
    ) {
        super();
    }
}

File: api/repositorio-principal/src/main/java/br/gov/df/emater/transporte/principal/PessoaFiltroDTO.java
package br.gov.df.emater.transporte.principal;

import java.util.Optional;
import java.util.Set;

import br.gov.df.emater.transporte.FiltroDTO;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@EqualsAndHashCode(callSuper = false)
public class PessoaFiltroDTO extends FiltroDTO {

	private static final long serialVersionUID = 1L;

	private Set<Optional<String>> cnpj;

	private Set<Optional<String>> cpf;

	private Set<Optional<String>> nome;

	private Set<Optional<String>> pesq;

	private Set<Optional<String>> siglaApelido;

}
